data/json/furniture.json

```diff
@@ -2655,7 +2655,34 @@
             { "item": "stick", "count": [8, 12] }
         ]
     }
-  },{
+    },{
+    "type" : "furniture",
+    "id" : "f_smoking_rack_active",
+    "name": "active smoking rack",
+    "description": "A special rack designed to smoke food for better preservation and taste.",
```

Comment:
Maybe add one additional line like"it is currently lit" to inform the player better?

Reply:
Ok. Asap.